drivers/spi/spi_rpi_pico_pio.c

```diff
@@ -206,10 +205,13 @@ static inline uint32_t spi_pico_pio_sm_get32(PIO pio, uint sm)
 	return *rxfifo;
 }
 
-static inline int spi_pico_pio_sm_complete(struct spi_pico_pio_data *data)
+static inline int spi_pico_pio_sm_complete(const struct spi_pico_pio_config *dev_cfg,
+					   struct spi_pico_pio_data *data)
```

Comment:
It doesn't seem necessary to pass in the `dev_cfg`, or am I missing something?
```suggestion
static inline int spi_pico_pio_sm_complete(PIO pio, struct spi_pico_pio_data *data)
```